migrations/20161123120702_add-update-contact-details-table.js

```diff
@@ -0,0 +1,18 @@
+exports.up = function (knex, Promise) {
+  return knex.schema.createTable('UpdateContactDetails', function (table) {
```

Comment:
prefix to be clear it's relationship with existing tables, EligibilityVisitorUpdateContactDetail (singular for DB tables)